README.md

```diff
@@ -117,4 +117,16 @@ Prerequisite:
 
 Run command:
 
-- `mvn exec:java -Dexec.mainClass="kx.examples.TypesMapping"`
\ No newline at end of file
+- `mvn exec:java -Dexec.mainClass="kx.examples.TypesMapping"`
+
+### Subscriber
+
+Example app that subscribes to real-time updates from a table that is maintained in KDB+. 
+
+Prerequisite: 
+
+- a KDB+ server running on port 5010 on your machine. The instance must have the .u.sub function defined. See this [knowledge base article](https://code.kx.com/q/kb/publish-subscribe/) for details.
```

Comment:
a KDB+ server (tickerplant process) running on port 5010 on your machine. The example of presumed `.u.sub` is found in [KxSystems/kdb-tick](https://github.com/KxSystems/kdb-tick) which is a example tickerplant code. You can run this tickerplant process by
```
tick]$ q tick.q
```
The default port is set 5010 in the startup.